torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -80,14 +80,16 @@ def __init__(
             cache: if True, cache file handle to speed up repeated sampling
 
         Raises:
-            FileNotFoundError: if no files are found in ``root``
+            FileNotFoundError: if no files are found in ``paths``
```

Comment:
I think you need a blank line between Raises and versionchanged

Reply:
Fixed